packages/ui-components/src/RequestErrors.tsx

```diff
@@ -94,4 +94,23 @@ export class RequestErrors {
       buttons: [Dialog.okButton()]
     });
   }
+
+  /**
+   * Displays an error dialog containing a syntax error.
+   *
+   * @param fileType - the type of file that was being accessed when
+   * the error occurred
+   *
+   * @returns A promise that resolves with whether the dialog was accepted.
+   */
+  static syntaxError(
```

Comment:
this is not a request error... maybe just make this a utility function on PipelineEditorWidget? or make a more generic error class and make this part of a new class like RuntimeError or something like that...